From a79536112453b333c5dcbee1017316591466e5fd Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Fri, 20 Jan 2012 11:01:31 +0800
Subject: [PATCH 0935/2463] ENGR00172395-2 MX6Q_ARM2: add sgtl5000 audio
 driver

add sgtl5000 driver for MX6Q_ARM2 in config file

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/imx/Kconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/sound/soc/imx/Kconfig b/sound/soc/imx/Kconfig
index cc0e2ca..89eecb9 100644
--- a/sound/soc/imx/Kconfig
+++ b/sound/soc/imx/Kconfig
@@ -52,7 +52,8 @@ config SND_SOC_PHYCORE_AC97
 
 config SND_SOC_IMX_SGTL5000
 	tristate "SoC Audio support for i.MX boards with sgtl5000"
-	depends on I2C && (MACH_MX35_3DS || MACH_MX51_BABBAGE || MACH_MX6Q_SABRELITE)
+	depends on I2C && (MACH_MX35_3DS || MACH_MX51_BABBAGE \
+			|| MACH_MX6Q_SABRELITE || MACH_MX6Q_ARM2)
 	select SND_SOC_SGTL5000
 	select SND_MXC_SOC_MX2
 	help
-- 
1.7.7.4

